src/utils/subgraphAPI.ts

```diff
@@ -0,0 +1,67 @@
+import axios from "axios"
+
+const TBTC_SUBGRAPH_URL =
+  "https://api.thegraph.com/subgraphs/name/suntzu93/threshold-tbtc"
+
+type TransactionRawResponse = {
+  // Actually transacion hash where the tbtc tokens were minted.
+  id: string
+  timestamp: string
+  deposits: {
+    actualAmountReceived: string
+    user: {
+      // ETH address
+      id: string
+    }
+  }[]
+}
+
+const mapToRecentDepositData = (transaction: TransactionRawResponse) => {
+  const amount = transaction.deposits[0].actualAmountReceived
+  const address = transaction.deposits[0].user.id
+  const txHash = transaction.id
+  const timestamp = +transaction.timestamp
+
+  return {
+    amount,
+    address,
+    txHash,
+    timestamp,
+  }
+}
+
+const fetchRecenttBTCDeposits = async (numberOfDeposits = 4) => {
```

Comment:
Non-blocking:

I believe we should write `TBTC` (started with capital T) when it's in the middle of the word that is writtne in camelCase convention. Everywhere else we should stick to `tBTC`.

```suggestion
const fetchRecentTBTCDeposits = async (numberOfDeposits = 4) => {
```

Reply:
c0b941f